test/jekyll-material-icon-tag/index.md

```diff
@@ -5,3 +5,5 @@ layout: home
 # About
 
 This page test the {% icon check %} gem.
+It also supports different {% icon alarm_on two-tone %} styles
+{% icon aspect_ratio round %}.
```

Comment:
Where or how do we see that this works?

Reply:
There are tests that checks that it is outputting correctly.

Reply:
Yes, I would just like to see the change as well.